src/microbe_stage/systems/MicrobeAISystem.cs

```diff
@@ -809,7 +827,7 @@ private void FleeFromPredators(ref WorldPosition position, ref MicrobeAI ai, ref
     private void EngagePrey(ref MicrobeAI ai, ref MicrobeControl control, ref OrganelleContainer organelles,
         ref WorldPosition position, CompoundBag ourCompounds, in Entity entity, Vector3 target, bool engulf,
         float speciesAggression,
-        float speciesFocus, float speciesActivity, Random random)
+        float speciesFocus, float speciesActivity, Random random, float strain)
```

Comment:
Shouldn't strain be named currentStrain here as well to be consistent with the other method taking in a strain parameter?